validation/Jenkinsfile.harvester

```diff
@@ -104,7 +105,7 @@ node("harvester-vpn-1") {
 
                 dir("./validation/.ssh") {
                   def decodedRsa = new String(AWS_SSH_RSA_KEY.decodeBase64())
-                  writeFile file: JENKINS_RKE_VALIDATION, text: decodedRsa
+                  writeFile file: AWS_RSA_KEY_NAME, text: decodedRsa
```

Comment:
Does changing this back work? I ask because when Windows clusters were not working with Jenkins, I specifically had to put `JENKINS_RKE_VALIDATION` along with creating the parameter in Jenkins for it to work. I know this for Harvester and vSphere below, but making sure this doesn't break Windows tests.

Reply:
yeah it was causing an issue with the harvester job. AWS_RSA_KEY_NAME is now in the secrets manager. The original jenkinsfile didn't have an entry for JENKINS_RKE_VALIDATION so it was failing since the var didn't exist.

Reply:
Got it. Especially as release testing is next week, can you run a simple windows provisioning check before this gets merged?

Reply:
I don't have a great way to run windows on harvester right now, setting up the image is complicated 😓 I can run it for the vsphere one though